test/UnitTest/Microsoft.AspNetCore.OData.Test/Extensions/HttpRequestExtensions.cs

```diff
@@ -1,22 +1,13 @@
 ﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
 // Licensed under the MIT License.  See License.txt in the project root for license information.
 
-#if NETCORE
 using System.Net;
 using Microsoft.AspNet.OData.Extensions;
 using Microsoft.AspNet.OData.Interfaces;
 using Microsoft.AspNetCore.Http;
-#else
-using System.Net.Http;
-using Microsoft.AspNet.OData.Extensions;
-#endif
 
 namespace Microsoft.AspNet.OData.Test.Extensions
 {
-#if NETCORE
-    /// <summary>
```

Comment:
good to keep for AspNetCore?

Reply:
This file is separated into two files, one in the classic, one in the AspNetCore.

Reply:
Ok. Got it. Thanks.